docs/src/modules/java/pages/views.adoc

```diff
@@ -565,6 +565,36 @@ The following data types are supported, for their corresponding Protobuf types.
 
 |===
 
+==== Optional fields
+
+Fields in a Protobuf message that was not given a value is handled as [the default value](https://developers.google.com/protocol-buffers/docs/proto3#default) of the field data type.
```

Comment:
small grammar fix: either singular `A field` or `Any field`, or use `were` and `are` to match the plural.

```suggestion
Fields in a Protobuf message that were not given a value are handled as [the default value](https://developers.google.com/protocol-buffers/docs/proto3#default) of the field data type.
```